commands/operator-sdk/cmd/up/local.go

```diff
@@ -78,6 +80,16 @@ func upLocal(projectName string) {
 		args = append(args, extraArgs...)
 	}
 	dc := exec.Command(gocmd, args...)
+	c := make(chan os.Signal)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-c
+		err := dc.Process.Kill()
+		if err != nil {
+			os.Exit(1)
```

Comment:
maybe log the error with `cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to terminate the operator: %v", err))`